mantidimaging/gui/test/gui_system_base.py

```diff
@@ -44,6 +45,10 @@ def tearDown(self) -> None:
         QTest.qWait(SHORT_DELAY)
         self.assertDictEqual(self.main_window.presenter.model.datasets, {})
 
+        if leak_count := leak_tracker.count():
+            print("\nItems still alive:", leak_count)
+            leak_tracker.pretty_print(debug_init=False, debug_owners=False, trace_depth=5)
```

Comment:
This looks like it's just passing in defaults, would it work to simplify to `leak_tracker.pretty_print()`?

Reply:
I left them in to make it quick and easy to flick them on when debugging.